### PR TITLE
HL-83

### PR DESCRIPTION
**Pull request by bitcoinbrisbane**
[Link to ticket](https://dltx.atlassian.net/jira/software/projects/HL/boards/82/backlog?selectedIssue=HL-83&text=HL-83)

- Python script to settle markets
- Only do bets in the last 2hrs
- Check to see if they have been settled first
- Update readme on how to run the script

image

### CODE REVIEW

File: scripts/settle.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+load_dotenv()
+
+node = os.getenv('GOERLI_URL')
+web3 = Web3(Web3.HTTPProvider(node))
+
+
+def get_markets():
+    response = requests.get("https://horse.link/api/config")
+    data = response.json()
+    return data['markets']
```
Comment (bazmatic) [inline scripts/settle.py:16]: The config endpoint returns markets?

**Review by bazmatic — COMMENTED** [review]
If there have been a lot of bets in between runs, this approach will miss those. Would it not be better to start from a given point, settle some bets, then save the last bet settled to be the starting point for next time?

**Review by brettvniekerk — DISMISSED** [review]

**Comment by bitcoinbrisbane** [comment]
Will chagne to last 2hrs, instead of 50

**Review by OptimusOpus — APPROVED** [review]
LGTM, nice to read some python to mix things up :)

**Review by bazmatic — APPROVED** [review]

**Pull request merged**